### PR TITLE
add XMR support

### LINKED ISSUE #162: Add support for Monero (XMR) addresses in encoding library
Here is the encoding library: https://github.com/ensdomains/address-encoder

Example: https://github.com/ensdomains/address-encoder/pull/47/files

Discord for dev help: https://discord.com/invite/AskZbFx

Please provide a reference link so we can check you did the encoding correctly.

**We'd like to keep the encoding library as small as possible, so please be efficient in your use of space. This is particularly important for this bounty.**

We will likely be reviewing PRs and paying out bounties more or less on a weekly basis. No need to ping us to check your PR, we will definitely get to it.

Thanks for your help!

### PR DESCRIPTION
**Pull request by canokaue**
PR regarding:
https://gitcoin.co/issue/ensdomains/address-encoder/162/100023867

## Issue number

Closes #162 
## Description

Add Monero address support.
## List of features added/changed

Add XMR on:
src/index.ts
src/tests/index.test.ts
README.md

Already sorted by Cointype on Typescript files and alphabetically on README.md

NOTE: If you are adding new coin address support, please make sure to add a reference link to README so that reviewer can verify.
https://github.com/satoshilabs/slips/blob/master/slip-0044.md
https://web.getmonero.org/resources/moneropedia/address.html

## Screenshots (if appropriate):
Local test run
TravisCI

## Checklist:

- [x] My code follows the code style of this project.
- [x] My code implements all the required features.

### CODE REVIEW

File: src/__tests__/index.test.ts [hunk 1]
```diff
@@ -138,6 +138,11 @@ const vectors: Array<TestVector> = [
       },
     ],
   },
+  {
+    name: 'XMR',
+    coinType: 128,
+    passingVectors: [{ text: 'AXaXZjZGA3qhQRTCsyG5uFKr9HeShgVhTF', hex: '17ad5cac596a1ef6c18ac1746dfd304f93964354b5' }],
```
Comment (makoto) [inline src/__tests__/index.test.ts:144]: Where did you get this test address? Your reference https://web.getmonero.org/resources/moneropedia/address.html says `A raw Monero address is a set of 95 characters starting with a '4'. The Monero donation address, for instance, is 888tNkZrPN6JsEgekjMnABU4TBzc2Dt29EPAvkRxbANsAnjyPbb3iQ1YBRk1UXcdRsiKc9dhwMVgN5S9cQUiyoogDavup3H.` which looks quite different

**Comment by makoto** [comment]
Hi, please let me know if you want to continue working on this (and other open PRs I commented on last week) or you want me to pass on to someone else.

**Comment by canokaue** [comment]
Hi @makoto , thanks for the heads up. Yes, I'm still active on the issue, you can expect updates on my open PRs this week still.

**Comment by makoto** [comment]
Actually there is a duplicate PR with https://github.com/ensdomains/address-encoder/pull/210 which has more correct and size efficient implementation so I will be merging that. You can continue working on other PRs you submitted as I don't see any duplicate PRs.

**Pull request closed without merging**